src/us_reverse_geo/Result.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A candidate is a possible match for an address that was submitted.<br>
+ *     A lookup can have multiple candidates if the address was ambiguous.
+ *
+ * @see "https://smartystreets.com/docs/cloud/us-reverse-geo-api#result"
+ */
+class Result {
+	constructor(responseData) {
+		this.coordinate = {};
+		if (respoonseData.coordinate !== undefined) {
+			this.coordinate.latitude = responseData.coordinate.latitude;
+			this.coordinate.longitude = responseData.coordinate.longitude;
+			this.coordinate.accuracy = responseData.coordinate.accuracy;
+			switch (responseData.coordinate.license) {
+				case 1:
+					this.coordinate.license = "SmartyStreets Proprietary";
```

Comment:
add `break` so it doesn't fall through to the `default` case
```suggestion
					this.coordinate.license = "SmartyStreets Proprietary";
					break;
```